src/Model/TransportModel/tsp1cnc1.f90

```diff
@@ -18,10 +20,16 @@ module GwtCncModule
   character(len=LENFTYPE) :: ftype = 'CNC'
   character(len=LENPACKAGENAME) :: text = '             CNC'
   !
-  type, extends(BndType) :: GwtCncType
+  type, extends(BndType) :: TspCncType
+
+    type(GweInputDataType), pointer :: gwecommon => null() !< pointer to shared gwe data used by multiple packages but set in mst
```

Comment:
If this is generic, shouldn't have any gwe variables.

Reply:
Addressed in 3bdd3d7